File: atividade2-stl1.lua
local t = tonumber(io.read())

for i=1, t do
    local n = tonumber(io.read())

    local dictionary = {}

    for j=1, n do
        local a, b = io.read():match("(%S+)%s+(%d+)")
        dictionary[a] = tonumber(b)
    end

    local d = tonumber(io.read()) -- tempo para fazer a atividade
    local topic = io.read()

    local time_taken = (dictionary[topic] or 100)

    if d >= time_taken then
        print("Case " .. i .. ": Ufa!")
    else
        if time_taken <= d + 5 then
            print("Case " .. i .. ": Atrasado")
        else
            print("Case " .. i .. ": Deu ruim! Va trabalhar")
        end
    end

    -- processamento dos prazos
end

File: atividade2-stl1.jl

print("Teste 1")
t = parse(Int, readline())

print("Teste 2")
for i in 1:t

    n = parse(Int, readline())
    dictionary = Dict{String, Int}()

    for j in 1:n
        input = readline()
        parts = split(input)
        
        dictionary[parts[1]] = parse(Int, parts[2])
    end

    d = parse(Int, readline()) # dias para fazer o trabalho
    topic = readline()

    time = 0

    if(haskey(dictionary, topic))
        time = dictionary[topic]
    end

    if d >= time
        print("Case ", i, ": Ufa!")
    else
        if time <= d + 5
            print("Case ", i, ": Atrasado")
        else
            print("Case ", i, ": Deu ruim! Va trabalhar")
        end 
    end
end